packages/atomic/src/components/common/interface/bindings.ts

```diff
@@ -1,13 +1,34 @@
 import {HTMLStencilElement} from '@stencil/core/internal';
 import {i18n} from 'i18next';
 import {AnyEngineType} from './interface-common';
+import {AtomicCommonStoreData} from './store';
+
+export interface ExtensibleStencilStore<
```

Comment:
how about `CommonStencilStore`, or something with Base or Interface? 
just bc all interfaces are extensible, so it sounds a bit funny